Sources/TypeScriptStandAlone/Generator.swift

```diff
@@ -0,0 +1,138 @@
+//
+// github.com/screensailor 2022
+//
+
+import Lexicon
+import UniformTypeIdentifiers
+
+public enum Generator: CodeGenerator {
+    
+    // TODO: prefixes?
+    
+    public static let utType = UTType.sourceCode
```

Comment:
We might want to look at this `UTType` so that when we export it adds the `.ts` extension automatically.